Makefile

```diff
@@ -21,10 +21,10 @@ DOCKER_IMAGE_NIM_PARAMS ?= -d:chronicles_colors:none -d:insecure
 	all \
 	deps \
 	update \
-	wakusim \
-	wakunode \
-	wakunode2 \
-	wakuexample \
+	sim \
+	node \
+	node2 \
+	example \
```

Comment:
I'd call example example1 here to make it clear.

This means test refers to both, test1 to v1 and test2 to v2